### PR TITLE
add stage 3 blurb

### PR DESCRIPTION
**Pull request by codehag**

### CODE REVIEW

File: _includes/proposals.html [hunk 1]
```diff
@@ -3,6 +3,10 @@
   <h2 class="section-headline text-end">State of Proposals</h2>
   <div class="featurelist">
     <h3>Stage 3</h3>
+    <i>
+      Proposals that have been reviewed, implemented in at least one vendor, and are being considered as
```
Comment (ljharb) [inline _includes/proposals.html]: this is incorrect; zero implementations are required for stage 3.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: _includes/proposals.html (hunk 1 above)
Comment (rkirsling) [reply _includes/proposals.html]: On the other hand, "complete spec text" is [listed](https://tc39.github.io/process-document/) as a requirement.

**Review by xtuc — APPROVED** [review]

**Comment by codehag** [comment]
@ljharb does this look better?

**Review by ljharb — APPROVED** [review]
This is fine; it’s a bit more pedantic than this in reality - specifically, the committee also has to have agreed, in a meeting, that it’s stage 3.

**Pull request merged**